src/app/zap-templates/zcl/data-model/chip/access-control-cluster.xml

```diff
@@ -18,6 +18,7 @@ limitations under the License.
   <domain name="CHIP"/>
 
   <enum name="AccessControlEntryPrivilegeEnum" type="enum8">
+    <cluster code="0x0752"/> <!-- Joint Fabric Datastore -->
```

Comment:
is this intentional?

Reply:
Yes, JointFabricDatastore cluster needs to have access to this enum.